### ISSUE TITLE
Juicebox Opens a Blank Window in Linux

### ISSUE THREAD

**Issue by taylorjtamu** [aidenlab/Juicebox#863]
When attempting to launch Juicebox on Scientific Linux 7.7 with OpenJDK JRE 64-Bit 1.8.0, regardless of the version of Juicebox attempted it opens a completely blank window. There is a titlebar and a space below that but nothing else is drawn at all. 
On old enough versions (1.8.8 specifically in this test), there is a popup window with a titlebar that states "Update Information" and has a clickable (but invisible) hyperlink in it. Nothing is drawn other than the titlebar and title.

This started after we had to do a system reinstall to update the operating system.

I highly suspect that we're simply missing some dependency that I'm not aware of and am mostly trying to figure out what that might be. Does Juicebox have logs stored anywhere that would point at the dependency I might be missing? Is there a listing anywhere I could compare against?

**Comment by sa501428** [comment]
Are you using the jar? Have you tried running the GUI via command line?
java -jar juicebox.jar

**Comment by taylorjtamu** [comment]
Yes, sorry I didn't specify. This is using the jar:

java -jar Juicebox_1.11.08.jar
java -jar Juicebox_1.8.8.jar
etc

I just tested logging into the box directly (physically) and was able to get Juicebox to display properly. It seems to be an issue with trying to run it remotely over our XRDP setup. I tested with another Linux box remotely and had the same issue.
I tested additionally with another system that's running Scientific Linux 6.10 over the XRDP/remote connection and Juicebox displayed the GUI properly.

Is this a known problem? Are there any configurations that I should modify to allow this to work? Are there any logs from Juicebox that might be helpful in resolving this? Unfortunately, our users only have remote access to these systems as they are rack mounted and kept in a secure server room.

**Comment by sa501428** [comment]
Do you have an admin at your institution? @weiszd @nchernia any thoughts?

**Comment by sa501428** [comment]
Also this may be a better discussion for our forum: aidenlab.org/forum.html

**Comment by taylorjtamu** [comment]
Yes, I have admin and am the main administrator for these systems. Ah, I wasn't aware of the forum. i can move this there if it's better.
